contracts/HubPool.sol

```diff
@@ -60,8 +60,6 @@ contract HubPool is HubPoolInterface, Testable, Lockable, MultiCaller, Ownable {
         uint256 claimedBitMap;
         // Proposer of this root bundle.
         address proposer;
-        // Whether bond has been repaid to successful root bundle proposer.
-        bool proposerBondRepaid;
```

Comment:
Note: this wasn't being used outside of this PR. We were using unclaimedPoolRebalanceLeafCount in lieu of this variable, so I figured it made sense to remove.